Assets/UI/Scenes/OptionsGraphics.tscn

```diff
@@ -0,0 +1,102 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://Assets/UI/Themes/MenuTheme.tres" type="Theme" id=1]
+
+[sub_resource type="GDScript" id=1]
+script/source = "extends GridContainer
+
+func _ready():
+	$NameEdit.text = Global.player_name
+	# add language from Global
+	var idx = 0
+	var i = 0
+	for n in Global._lang_array:
```

Comment:
Invalid get index '_lang_array' (on base: 'Node (Global.gd)').

Reply:
I mixed up the upload. :-/